src/plugins/typescript/index.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2017-present Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain
+ * a copy of the license in the LICENSE and PATENTS files in the root of this
+ * repository.
+ */
+
+import { Application, ContainerReflection, ReflectionKind } from "typedoc";
+import { ITypescriptPluginData } from "../../client";
+import { ICompiler, IFile, IPlugin } from "../plugin";
+import { isInternal, visitorExportedClass, visitorExportedInterface } from "./visitors";
```

Comment:
these are the only two exported visitor methods, as they're the "entry points" to the documentation.